administrator/modules/mod_version/services/provider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  mod_version
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+ \defined('_JEXEC') or die;
```

Comment:
```suggestion
\defined('_JEXEC') or die;
```
Fix PHPCS failing in drone due to space at beginning of this line.